website/common/script/ops/openMysteryItem.js

```diff
@@ -24,6 +24,7 @@ module.exports = function openMysteryItem (user, req = {}, analytics) {
   if (mysteryItems.length === 0) markNotificationAsRead(user);
 
   item = cloneDeep(content.gear.flat[item]);
+  item.text = content.gear.flat[item.key].text();
```

Comment:
shouldn't this be passed the language?

Reply:
I assumed it would get the language automatically, but you're probably right.